### PR TITLE
Explicitly enable CGO and release v0.2.6

### PR DESCRIPTION
**Pull request by mprahl**
Addresses CVE-2023-3089

### CODE REVIEW

**Review by dhaiducek — APPROVED** [review]

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Should the builder image be 1.20?

**Comment by dhaiducek** [comment]
Oops, actually the 1.20 would require more upgrades--I didn't notice this was `release-0.2`, so upgrading the builder image doesn't make sense unless we're upgrading everything else also.

File: Makefile [hunk 1]
```diff
@@ -323,6 +323,10 @@ import-manifests: kustomize
 		$(KUSTOMIZE) build --load-restrictor LoadRestrictionsNone $(IMPORT_MANIFESTS_PATH)/config/overlays/mutation -o $(GATEKEEPER_MANIFEST_DIR); \
 	fi
 
+	sed -i '73,80d' $(GATEKEEPER_MANIFEST_DIR)/rbac.authorization.k8s.io_v1_clusterrole_gatekeeper-manager-role.yaml
```
Comment (dhaiducek) [inline Makefile:326]: Specifying line numbers is prone to error if there's an update. Could we use a `yq` command here instead? Maybe something like this?
```
yq e '.rules[] |= del(select(.resources[] == "podsecuritypolicies"))' -i $(GATEKEEPER_MANIFEST_DIR)/rbac.authorization.k8s.io_v1_clusterrole_gatekeeper-manager-role.yaml
```
Please also add a brief comment here on why it's necessary.

Comment (mprahl) [reply Makefile]: Neat!

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**